MobilliumDateFormatter/Classes/Date+BoolOperations.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Date+BoolOperations.swift
+//  MobilliumDateFormatter
+//
+//  Created by Mehmet Salih Aslan on 17.03.2021.
+//
```

Comment:
We can add copyright to here
```suggestion
// Copyright © 2021 Mobillium. All rights reserved.
```